core/controllers/dashboard.py

```diff
@@ -243,11 +243,28 @@ def _round_average_ratings(rating):
             last_week_stats['average_ratings'] = (
                 _round_average_ratings(last_week_stats['average_ratings']))
 
+        subscriber_ids = subscription_services.get_all_subscribers_of_creator(
+            self.user_id)
+        subscribers_settings = user_services.get_users_settings(subscriber_ids)
+        subscribers_list = []
+        for index, subscriber_settings in enumerate(subscribers_settings):
+            subscriber_summary = {}
```

Comment:
In line with @prasanna08's comment below, it would be better to at least do:

    subscriber_summary = {
        'subscriber_picture_data_url': (
            subscriber_settings.profile_picture_data_url),
         ...
    }

Otherwise it's a bit like you're redefining subscriber_summary again and again.